### ISSUE TITLE
Hostnames in graph differ between reports and single metrics

### ISSUE THREAD

**Issue by cwestvpac** [ganglia/ganglia-web#223]
When I look at a single metric, say load_one, load_five and load_fifteen the hostname is short (not fully qualified - which I prefer). However, when loading a report, say the load_all_report the hostname is now fully qualified. 

It seems to be the same for all reports (they are fully qualified). 

I feel the graphs should be as similar as possible, and would vote for the short hostnames to be used in the reports.

Note: This is has been seen in ganglia-web 3.5.12 and 3.3.8

**Comment by 00gavin** [comment]
Agreed.  I just updated to 3.7.1 and am setting '$strip_domainname = true;' in conf.php and still getting full, not short, hostnames in the aggregated graph.
